blocklayered.php

```diff
@@ -1041,7 +1041,11 @@ public function indexUrl($ajax = false, $truncate = true)
 				foreach ($attribute as $param)
 				{
 					$selected_filters = array();
-					$link = '/'.str_replace($this->getAnchor(), '_', Tools::link_rewrite($param['name'])).$this->getAnchor().str_replace($this->getAnchor(), '_', Tools::link_rewrite($param['value']));
+					if ($param['type'] == 'category') {
+						$link = '/' . $param['id_parent'] . $this->getAnchor() . str_replace($this->getAnchor(), '_', Tools::link_rewrite($param['name'])) . $this->getAnchor() . str_replace($this->getAnchor(), '_', Tools::link_rewrite($param['value']));
+					} else {
+						$link = '/' . str_replace($this->getAnchor(), '_', Tools::link_rewrite($param['name'])) . $this->getAnchor() . str_replace($this->getAnchor(), '_', Tools::link_rewrite($param['value']));
+					}
```

Comment:
@Quetzacoalt91 could you please review it and merge it? 
it is supposed to be merged since 1.6.1.14 but no one check it :confused: